control-plane/p4RuntimeSerializer.cpp

```diff
@@ -792,9 +811,9 @@ class P4RuntimeSymbolTable {
     P4SymbolSuffixSet suffixSet;
 };
 
-/// A serializer which translates the information available in the P4 IR into a
+/// An analyzer which translates the information available in the P4 IR into a
 /// representation of the control plane API which is consumed by P4Runtime.
-class P4RuntimeSerializer {
+class P4RuntimeAnalyzer {
```

Comment:
The fact that we need to deal in both the normal IR representation of the P4 program and with the evaluated version. The latter doesn't really work with the normal visitor idioms we use, unfortunately; that's been the problem since the beginning.